editor/plugins/editor_preview_plugins.cpp

```diff
@@ -467,15 +467,6 @@ Ref<Texture> EditorScriptPreviewPlugin::generate(const RES &p_from) {
 	Color text_color = EditorSettings::get_singleton()->get("text_editor/highlighting/text_color");
 	Color symbol_color = EditorSettings::get_singleton()->get("text_editor/highlighting/symbol_color");
 
-	if (EditorSettings::get_singleton()->get("text_editor/theme/color_theme") == "Adaptive") {
-		Ref<Theme> tm = EditorNode::get_singleton()->get_theme_base()->get_theme();
-
-		bg_color = tm->get_color("text_editor/theme/background_color", "Editor");
```

Comment:
They should likely be fixed and not removed (`"text_editor/highlighting/background_color"`): https://github.com/godotengine/godot/blob/9f479f096cb50fa8d1215e68c262f110116114ef/editor/editor_themes.cpp#L1069

Reply:
CC @djrm @volzhs

Reply:
There is no Adaptive theme colors are existed, only the setting colors, which already setted on previous lines. The PR from poke1024 corrupt the script preview cache now for all peoples, it should be fixed ASAP..

Reply:
However I could change it to "text_editor/highlighting/" instead "text_editor/theme"

Reply:
> The PR from poke1024 corrupt the script preview cache now for all peoples, it should be fixed ASAP..

It's the master branch, there's no hurry.. If previews are broken for a couple of days and fixed automatically later on, there's no harm done.